content/standards/360giving/index.html.md

```diff
@@ -1,14 +1,15 @@
 ---
-title: 360Giving Schema
+name: 360Giving Schema
+contraction: 360GS
+specification_url: https://standard.threesixtygiving.org/en/latest/reference/#spreadsheet-format
 tags:
 - domain-specific
 - payments
-organisation: three-sixty-giving
-name: 360Giving Schema
-status: review
-dateAdded: 2021-02-02
-dateUpdated: 2021-02-02
+maintainer_id: three-sixty-giving
 licence_id: CC-BY-4.0
+endorsement_status_events:
+- status: review
+  date: "2021-02-02"
```

Comment:
I would suggest we use unquoted dates as defined by the [YAML spec](https://yaml.org/spec/1.2.2/#example-timestamps). That said I haven't checked whether the yaml parser we use has any issues with it.

Reply:
Perhaps something to consider for the future is having an agreed YAML linting process to converge in style?

Reply:
I believe this came out of issues where not using quotes caused problems, but I can't remember :sweat_smile:

And agreed, that would probably be ideal - I'll raise an issue